cuckoo/cuckoo_result.py

```diff
@@ -1710,10 +1708,8 @@ def _write_tree_and_hashes(stripped_process_tree: List[Dict], process_tree_hashe
             else:
                 file.write(str(stripped_process_tree[index]).replace('\\\\', '\\').replace('\'', '\"') + '\n')
 
-    file.close()
 
-
-def _remove_safe_roots(process_tree: List[Dict[str, Any]], process_tree_hashes: List[str], safe_process_tree_hashes: Set[str]) -> None:
+def _remove_safe_roots(process_tree: List[Dict[str, Any]], process_tree_hashes: List[str], safe_process_tree_hashes: Dict[str, Any]) -> None:
```

Comment:
Since these methods are not in a class but rather in a module, they require 2 blank lines between them